target/linux/ath79/generic/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -3,200 +3,12 @@
-	ubnt,litebeam-ac-gen2|\
-	ubnt,nanobeam-ac|\
-	ubnt,nanobeam-ac-gen2|\
-	ubnt,nanobeam-ac-xc|\
-	ubnt,nanostation-ac|\
-	ubnt,nanostation-ac-loco|\
-	ubnt,powerbeam-5ac-500|\
-	ubnt,powerbeam-5ac-gen2|\
-	ubnt,rocket-5ac-lite|\
-	ubnt,unifiac-pro|\
-	yuncore,a770|\
-	yuncore,xd3200)
-		caldata_extract "art" 0x5000 0x844
-		;;
-	avm,fritz1750e|\
-	avm,fritzdvbc)
-		caldata_extract "urlader" 0x198a 0x844
-		;;
-	devolo,dvl1200e|\
-	devolo,dvl1200i|\
-	devolo,dvl1750c|\
-	devolo,dvl1750e|\
-	devolo,dvl1750i|\
-	devolo,dvl1750x)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary art 0x0) -1)
-		;;
-	engenius,ecb1200|\
-	engenius,ecb1750)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(mtd_get_mac_ascii u-boot-env athaddr)
-		;;
-	engenius,ews511ap)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(cat /sys/class/net/eth0/address) 1)
-		;;
-	extreme-networks,ws-ap3805i)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(mtd_get_mac_ascii cfg1 RADIOADDR0)
-		;;
-	glinet,gl-ar750)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary art 0x0) 1)
-		;;
-	iodata,wn-ac1167dgr|\
-	iodata,wn-ac1600dgr2|\
-	sitecom,wlr-7100|\
-	zyxel,nbg6616)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_ascii u-boot-env ethaddr) 1)
-		;;
-	nec,wg800hp)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(mtd_get_mac_text board_data 0x880)
-		;;
-	ocedo,koala|\
-	ocedo,ursus)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(mtd_get_mac_binary art 0xc)
-		;;
-	openmesh,a40|\
-	openmesh,a60|\
-	openmesh,mr1750-v1|\
-	openmesh,mr1750-v2|\
-	openmesh,om5p-ac-v2)
-		caldata_extract "ART" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(cat /sys/class/net/eth0/address) 16)
-		;;
-	qihoo,c301)
-		caldata_extract "radiocfg" 0x5000 0x844
-		ath10k_patch_mac $(mtd_get_mac_ascii devdata wlan5mac)
-		;;
-	tplink,archer-a7-v5|\
-	tplink,archer-c2-v3|\
-	tplink,archer-c7-v4|\
-	tplink,archer-c7-v5|\
-	tplink,archer-c25-v1|\
-	tplink,tl-wr902ac-v1)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary info 0x8) -1)
-		;;
-	tplink,archer-c5-v1|\
-	tplink,tl-wdr7500-v3)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary u-boot 0x1fc00) -1)
-		;;
-	tplink,archer-d50-v1)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary romfile 0xf100) 2)
-		;;
-	tplink,archer-d7-v1|\
-	tplink,archer-d7b-v1)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary romfs 0xf100) 2)
-		;;
-	tplink,re350k-v1)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary info 0x8) 2)
-		;;
-	tplink,re355-v1|\
-	tplink,re450-v1)
-		caldata_extract "art" 0x5000 0x844
-		ath10k_patch_mac $(macaddr_add $(cat /sys/class/net/eth0/address) -2)
-		;;
```

Comment:
think we are missing these? and we are setting the addr from the info partition (from rxc5x dtsi?)

Reply:
The `eth0` MAC is fetched from info mtd. So, we can configure it directly in dts.
https://github.com/openwrt/openwrt/blob/3a073a021286bef3fed0ca7884872c5890420d60/target/linux/ath79/dts/qca9558_tplink_rex5x.dtsi#L173